cli/run_testenv.fish

```diff
@@ -23,7 +23,11 @@ set SCRIPT_DIR (dirname (realpath (status -f)))
 # In Python we trust (aka shell's tempfile&mktemp doesn't work on all platforms)
 set SOURCE_FILE (python -c "import tempfile; print(tempfile.mkstemp()[1])")
 
+echo ">>> " "cargo run --package parsec_cli --features testenv run-testenv --main-process-id $fish_pid --source-file \"$SOURCE_FILE\" $argv"
 eval cargo run --package parsec_cli --features testenv run-testenv --main-process-id $fish_pid --source-file "$SOURCE_FILE" $argv; or exit $status
+
+sed --in-place --regexp-extended 's/^export (\w+)=/set --export \1 /' $SOURCE_FILE
```

Comment:
Please add a comment to explain what this does

Reply:
done :scroll: